.github/workflows/continous-integration.yml

```
@@ -1,4 +1,4 @@
-name: Test
+name: CI
```

Can we make the Docker and OS names more consistent?
And start both files with the same word?

```suggestion
name: Test on Docker
```